src/transaction/log_record.hpp

```diff
@@ -320,6 +322,9 @@ struct log_rec_sysop_end
       bool is_sysop_postpone;	/* true if run postpone is used during a system op postpone, false if used during
 				 * transaction postpone */
     } run_postpone;		/* run postpone info */
+    // *INDENT-OFF*
```

Comment:
what about moving it before the union ?
(easier debugging)

Reply:
It is part of the union; each type of sysop end has different data.